### PR TITLE
Added Turkish language support.

### PR DESCRIPTION
**Pull request by Draconis-25**

### CODE REVIEW

**Comment by Draconis-25** [comment]
Just to inform, I havent updated the README file.

File: lorien/Assets/I18n/tr.txt [hunk 1]
```diff
@@ -0,0 +1,139 @@
+LANGUAGE_NAME Turkish
```
Comment (mbrlabs) [inline lorien/Assets/I18n/tr.txt:1]: This is supposed to be written in the language itself...so the turkish word for the word "Turkish" :)

**Review by mbrlabs — CHANGES_REQUESTED** [review]
Thank you for your contribution, looks good! Just a little nitpick.

File: lorien/Assets/I18n/tr.txt (hunk 1 above)
Comment (Draconis-25) [reply lorien/Assets/I18n/tr.txt]: Thank you for informing. I've fixed the issue.

**Comment by Draconis-25** [comment]
You are welcome. Readme is still not updated though.

**Comment by mbrlabs** [comment]
Oh right, i just updated it.

**Pull request merged**